themes/deis/base.html

```diff
@@ -27,27 +27,7 @@
         <![endif]-->
     <div class="container">
         <div class="row">
-            <div class="span3 column_calc">
```

Comment:
Yep! We will get a fresh paint of coat soon, but wanted to increase the readability now, this gives the content much more room:

![2016-05-18 at 09 06](https://cloud.githubusercontent.com/assets/28873/15366164/e44ab7b0-1cd7-11e6-920d-40f8aa384dd1.png)